crates/topos-sequencer-subnet-runtime/src/certification.rs

```diff
@@ -19,6 +18,8 @@ pub struct Certification {
     signing_key: Vec<u8>,
     /// Optional synchronization from particular block number
     pub start_block: Option<u64>,
+    /// Latest BLOCK_HISTORY_LENGTH blocks kept in memory
```

Comment:
```suggestion
    /// We keep [`BLOCK_HISTORY_LENGTH`] blocks in memory to check that the we process the block in the right order.
```

…or something that explains what this actually does.

Reply:
Updated comment, this one was outdated